tests/utils/skipif.php

```diff
@@ -10,6 +10,18 @@
 require_once __DIR__ . '/basic.inc';
 require_once __DIR__ . '/tools.php';
 
+/**
+ * Disables SKIPIF caching (PHP 8.1+).
+ */
+function skip_if_nocache()
```

Comment:
Nit: this suggests that this method skips tests if there's no cache. disable_skipif_caching seems more appropriate, but I defer to your judgment.

Reply:
That thought did occur to me when writing this. I agree that `disable_skipif_caching` is a much better name.